src/Simulation/CsharpGeneration/SimulationCode.fs

```diff
@@ -1032,7 +981,7 @@ module SimulationCode =
                 | false, _ -> startLine
             ``attribute`` None (ident "SourceLocation") [ 
                 ``literal`` sp.SourceFile.Value 
-                ``ident`` "OperationFunctor" <|.|> ``ident`` bodyName 
```

Comment:
Again, add these ticks back so its consistent that any method that generates C# uses the backticks.